source/documentation/data-docs/curated-databases-docs/databases.md

```diff
@@ -1,9 +1,17 @@
 # Databases
 
-The data engineering team curate and maintain databases on the Analytical Platform that can be made accessible to users. All of our databases are currently deployed and accessible using Amazon Athena. We currently support users being able to query these databases via the [Athena SQL workbench](#amazon-athena), [R and Python](#dbtools).
+The data engineering team curate and maintain databases on the Analytical Platform that can be made accessible to users. All of our current databases are deployed and accessible using Amazon Athena. AP users can query these databases via the [Athena SQL workbench](https://eu-west-1.console.aws.amazon.com/athena/home?region=eu-west-1#query), and in R and Python using [dbtools](https://github.com/moj-analytical-services/dbtools#dbtools) and [pydbtools](https://github.com/moj-analytical-services/pydbtools#pydbtools) respectively.
```

Comment:
I am not sure it is worth giving the link to Athena as AP users probably should go via CP. You could just point to the Athena documentation of this guidance (which walks through how to get to the workbench) instead?